src/main/java/org/jboss/xavier/integrations/migrationanalytics/business/VMWorkloadInventoryCalculator.java

```diff
@@ -56,6 +61,15 @@ private VMWorkloadInventoryModel createVMWorkloadInventoryModel(Map vmStructMap,
         model.setVersion(readValueFromExpandedEnvVarPath(VERSIONPATH, vmStructMap));
         model.setHost_name(readValueFromExpandedEnvVarPath(HOSTNAMEPATH, vmStructMap));
 
+        Date scanRunDate;
+        try {
```

Comment:
Running some test in CI, i figured out that `data_collected_on` is a field that we could read once in `calculate` method instead of reading it every time for every VM.
This helps the case when the field is available (since we read it just once) and maybe even more in the case when it's not available and we have to manage two exceptions each time we create a VM instead of just managing the 2 exceptions once.

Reply:
ok